pyproject.toml

```diff
@@ -148,7 +148,7 @@ sso = ["social-auth-core"]
 
 [tool.poetry.dev-dependencies]
 # Code style enforcement
-black = "~22.6.0"
+black = "~22.8.0"
```

Comment:
I don't see anything in the release-notes for black that raises any concern to me. :+1: